candi local.cfg: add netcdf

tjhei opened the pull request:
part of #5447

MFraters reviewed (approved):
Looks good, but shouldn't netcdf also be added to the `doc/sphinx/user/install/local-installation/using-candi.md` optional packages which may be required in the future? Or do we just want to update the text to require both?

tjhei commented:
> optional packages which may be required in the future? Or do we just want to update the text to require both?

We are talking about making sundials and netcdf mandatory in the near future. But you are right. I think I addressed your comment.

bangerth commented:
The indent check fails.

bangerth commented on doc/sphinx/user/install/local-installation/using-candi.md:
```diff
@@ -34,6 +32,15 @@ class="smallcaps">p4est</span> and Trilinos.
+    
+    If you want to work without the `local.cfg`, be aware that you should enable
+    - p4est
+    - trilinos
+    - hdf5 (optional)
+    - netcdf (optional)
+    - sundials (optional)
```
I couldn't quite figure out what the "(optional)" part here means. You're suggesting in the text above that one should enable these. If that's your suggestion, then just remove the qualification, or at least explains what happens if one takes the optional at face value and doesn't do it.

tjhei replied:
Fair enough. Updated.
